File: back_end/src/main/java/com/Xplored/Xplored/Model/Place/PlaceRequest.java
package com.Xplored.Xplored.Model.Place;

public class PlaceRequest {
    public String name;
    public String description;
    public double lat;
    public double lng;
    public String addressFull;
    public String coverImageUrl; // Added this just in case
    public Long categoryId;
    public String authorId; // This corresponds to the user's email
    public String status;
}

File: front_end/app/src/main/java/pt/iade/ei/xplored/models/PlaceRequest.kt
package pt.iade.ei.xplored.models

data class PlaceRequest(
    val name: String,
    val description: String,
    val lat: Double,
    val lng: Double,
    val addressFull: String = "",
    val postalCode: String = "",
    val categoryId: Long = 1,
    val status: String = "pending"
)
